docs/usage/shoot_status.md

```diff
@@ -121,19 +121,23 @@ The Shoot status also contains information about the last occurred error(s) (if
-- `ERR_INFRA_UNAUTHORIZED` - Indicates that the last error occurred due to the server understanding the request but refusing to authorize it. It is classified as a non-retryable error code.
-- `ERR_INFRA_QUOTA_EXCEEDED` - Indicates that the last error occurred due to infrastructure quota limits. It is classified as a non-retryable error code.
-- `ERR_INFRA_RATE_LIMITS_EXCEEDED` - Indicates that the last error occurred due to exceeded infrastructure request rate limits.
-- `ERR_INFRA_DEPENDENCIES` - Indicates that the last error occurred due to dependent objects on the infrastructure level. It is classified as a non-retryable error code.
-- `ERR_RETRYABLE_INFRA_DEPENDENCIES` - Indicates that the last error occurred due to dependent objects on the infrastructure level, but the operation should be retried.
-- `ERR_INFRA_RESOURCES_DEPLETED` - Indicates that the last error occurred due to depleted resource in the infrastructure.
-- `ERR_CLEANUP_CLUSTER_RESOURCES` - Indicates that the last error occurred due to resources in the cluster that are stuck in deletion.
-- `ERR_CONFIGURATION_PROBLEM` - Indicates that the last error occurred due to a configuration problem. It is classified as a non-retryable error code.
-- `ERR_RETRYABLE_CONFIGURATION_PROBLEM` - Indicates that the last error occurred due to a retryable configuration problem. "Retryable" means that the occurred error is likely to be resolved in a ungraceful manner after given period of time.
-- `ERR_PROBLEMATIC_WEBHOOK` - Indicates that the last error occurred due to a webhook not following the [Kubernetes best practices](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#best-practices-and-warnings).
+Known error codes and their classification are:
+
+| Error code                            | User error | Description                                                                                         |
+| ------------------------------------- | :--------: | --------------------------------------------------------------------------------------------------- |
+| `ERR_INFRA_UNAUTHENTICATED`           | true       | Indicates that the last error occurred due to the client request not being completed because it lacks valid authentication credentials for the requested resource. It is classified as a non-retryable error code. |
+| `ERR_INFRA_UNAUTHORIZED`              | true       | Indicates that the last error occurred due to the server understanding the request but refusing to authorize it. It is classified as a non-retryable error code. |
+| `ERR_INFRA_QUOTA_EXCEEDED`            | true       | Indicates that the last error occurred due to infrastructure quota limits. It is classified as a non-retryable error code. |
+| `ERR_INFRA_RATE_LIMITS_EXCEEDED`      | false      | Indicates that the last error occurred due to exceeded infrastructure request rate limits. |
```

Comment:
You have mentioned `ERR_INFRA_RATE_LIMITS_EXCEEDED` as false in `User error` also 
`**Please note:** Errors classified as `User error: true` do not require a Gardener operator to resolve but can be remediated by the user (e.g. by refreshing expired infrastructure credentials).`

Above line says only `User error: true` do not require a Gardener operator, I wonder what operator can do in case of `ERR_INFRA_RATE_LIMITS_EXCEEDED` or `ERR_RETRYABLE_INFRA_DEPENDENCIES`.

Reply:
For `ERR_INFRA_RATE_LIMITS_EXCEEDED`:
It is true that an operator probably has very little chance of influencing it. However, I would ask it the other way around - what should a user do about it? After all, the reconciliation of the `infrastructure` resource is part of the Gardener stack and cannot be influenced by the user.

For `ERR_RETRYABLE_INFRA_DEPENDENCIES`: 
As long as it is labeled as `retryable` I would consider it a temporary issue and therefore nothing a user should check (yet).

In general, I believe the classification is primarily helping users to identify issues they can potentially tackle themselves. If we change the classification here, we should also consider changing it for the dashboard.

Reply:
ping @acumino

Reply:
ping @acumino

Reply:
> In general, I believe the classification is primarily helping users to identify issues they can potentially tackle themselves. If we change the classification here, we should also consider changing it for the dashboard.

Yes but, I don't know how many will read this doc before pinging a support person(SRE), and even if somebody reads they can come back to operator for `ERR_INFRA_RATE_LIMITS_EXCEEDED` and  `ERR_RETRYABLE_INFRA_DEPENDENCIES` error code.
May be you can enhance the note.